hieradata/common.yaml

```diff
@@ -79,6 +79,33 @@ kubernetes::master_nodes:
     - autocrat
     - deadlock
 
+# Kubectl version
+kubectl_package_version: 1.19.4-00
+
+# Kubernetes config (the new setup)
+kube::kubernetes_version: 1.19.4
```

Comment:
This is probably to avoid conflict with the current class name(?). In any case is there a plan for transitioning from the current setup to this?

Reply:
Migrate all existing kubernetes services to the new cluster, and you can just delete all the code for the old configuration.